rskj-core/src/test/java/co/rsk/peg/BridgeSupportProcessFundsMigrationTest.java

```diff
@@ -184,10 +184,10 @@ private void test_processFundsMigration(
 
         long federationCreationBlockNumber = 5L;
         long federationInMigrationAgeHeight = federationCreationBlockNumber +
-            bridgeConstants.getFederationActivationAge() +
+            bridgeConstants.getFederationActivationAge(activations) +
```

Comment:
Looks like we are missing tests in this class also, we need to check that the federation is activated later after fingerroot

Reply:
Same here: https://github.com/rsksmart/rskj/blob/increase-new-fed-activation-age/rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java#L69-L105

Reply:
Done